test/js/specs/happy.js

```diff
@@ -1,16 +1,50 @@
 const path = require('path')
+const expect = require('chai').expect;
 import {describe, it} from '../utils/mochaw'
 
 const options = {
   pageObjects: ['DocumentSelection', 'Welcome', 'DocumentUpload']
 }
 
-describe('Happy Paths',options, ({driver,pageObjects}) => {
+const localhostUrl = 'https://localhost:8080/'
+
+describe('Happy Paths',options, ({driver,$,pageObjects}) => {
   const {documentSelection, welcome, documentUpload} = pageObjects
 
+  //Welcome Screen Tests
+  const copy = welcome.copy()
+
+  it('test website title', async () => {
```

Comment:
Not used to the JS way of doing things, but is it normal to include `test` in the spec description?

In my non-informed option (and i'm biased from ruby rspec) i would say:

```
it('contains the website title', ....)
```

or even better:

```
it('contains the title', ....)
```

Reply:
I have checked a few implementations and there are some implementations that use the word `should` like `it('should contain the title', ....)`

Reply:
yep, i 100% agree with the `should` :+1: 

Good investigation :+1: